certdb/testdb/testdb.go

```diff
@@ -0,0 +1,61 @@
+package testdb
+
+import (
+	"database/sql"
+
+	_ "github.com/lib/pq"           // register postgresql driver
+	_ "github.com/mattn/go-sqlite3" // register sqlite3 driver
+)
+
+// SQLiteDB returns a SQLite db instance for certdb testing.
+func SQLiteDB(dbpath string) *sql.DB {
```

Comment:
I think this function should come after the `const` declaration for consistency with other files